src/managers/PromptsManager.ts

```diff
@@ -194,21 +200,29 @@ export class PromptsManager {
       Log.debug('Not showing slidedown permission message because styles failed to load.');
       return;
     }
-    const slideDownOptions: SlidedownPermissionMessageOptions =
-      MainHelper.getSlidedownPermissionMessageOptions(OneSignal.config.userConfig.promptOptions);
+    const slidedownOptions =
+      MainHelper.getSlidedownOptions(OneSignal.config.userConfig.promptOptions);
 
     if (!this.eventHooksInstalled) {
       this.installEventHooksForSlidedown();
     }
 
-    OneSignal.slidedown = new Slidedown(slideDownOptions);
+    // TO DO: iterate through each config in slidedownOptions and mount individually...
+    // existing slidedown types: push, category. in current state, if category slidedown
+    // is configured we use that config. in future, we should support both if configured
+    // simultaneously
+    const slidedownPromptOptions = PromptsHelper
```

Comment:
Need to confirm if both Category and Push can be on at the same time. Check in dashboard UI AND custom code in the current in the wild config

Reply:
It is not possible.